### DISCUSSION TITLE
Allow expanding focused windows

### DISCUSSION THREAD

**Discussion by michaelmoreno** [ianyh/Amethyst#1303] [Ideas]
It seems that Amethyst only allows you to expand/shrink the main pain. Other tiling managers allow you to shrink/expand whatever window you're focusing on. This is much more convenient than having to go to the window you want to expand, make it the main pain, then expand it.

**Comment by ianyh** [comment]
The basic tiling model is (intentionally) rather simplified. That said, this is mostly a matter of existing layouts not supporting it rather than the system itself. Instead of a proliferation of compiled layouts I opted for keeping the basic model simple with the capability for externally defined layouts.
